docs/requirements.md

```diff
@@ -1,16 +1,59 @@
+* As a user, I want to know if URL(s) on my branch still have problematic performance in an absolute way, such as too many database queries, are too long for an acceptable web request, or eats up too much memory.
+
+## Workflow & CI
+
+* As a user, I want to easily start a job from a Github PR without many steps beyond knowing which URL(s) to test.
+* As a user, I want to have a small set of URLs that can be tested against master on most (but not all) of my team's PRs.
+* As a code reviewer, I want to see iterative history that performance was later improved on a PR that initially had poor performance.
+
+## Social Proof
+
+* As a user, I want my GitHub PR to display an up-to-date concise summary of performance status that links to details of the latest Perf Check job for the branch.
+* As a user, I'd like to share my results summary with someone in slack.
+
+## Error Handling
+
+* As a user, I want to know if everything on Perf Check CI is actually working right now.
+* As a Perf Check developer, I want to know if all specific components of the Perf Check CI system are functioning properly (db online, db last updated, queue status, target app bootable).
+* As the product owner, I want to communicate clearly to a user when a job failure or error was the result of problems with their branch or user supplied Job settings.
+* As a user, I want to know if the target application did not boot on my branch, but did boot on master.
+* As a user, I want to know if the URL 404'd or 500'd on the target app, see the error in a format I'm used to, and receive guidance on whether that was a result of my branch's changes or if master had the same problem.
+* As a user, I'd like to know when the error was the result of bugs/issues with the Perf Check CI stack that it's communicated to the Perf Check developers.
 
 ## Measurements and statistics
 
-* As a developer I want to store all measurements from a PerfCheck run so can process them later.
-* As a developer I want to compute summary statistics over measurements so I can use them for reporting.
+* As a product owner, I want to present very limited and clear "headline" guidance to users so they attend to and optimize "big picture" performance instead of being distracted by Perf Check implementation details.
+* As a Perf Check developer, I want to store all measurements from a Perf Check run so can process them later.
+* As a Perf Check developer, I want to compute summary statistics over measurements so I can use them for reporting.
+* As a user, I want to see a detailed breakdown of each request made.
+* As a user, I want to know variance of a group of requests and know how "normal" that variance is, so I feel comfortable trusting the summarized results and knowing whether the request profile is changed or unchanged.
+* As a user, I want to gain intuition about request performance variance via a visual representation of where the reference and branch requests fall on a shared timeline.
```

Comment:
Has anyone you spoke to actually voiced concerns over variance in the measurements? Should we summarize the last three stories?

* As a user, I want to see all benchmarked requests with an explanation of how they influence the Perf Check result so I can have more trust in the algorithm.

Reply:
I'm fine with conflating the last two. Seeing the individual requests seems important, as the CLI provides this and removing it would be confusing, I think.

Reply:
Rethinking this, I think variance is probably only really important to Perf Check developers. Presenting it to users might be friendly, assuming the context is provided (i.e. how normal the spread is). I'll come up with something that tries to be generic.

Reply:
Ah there is overlap here with 

* As a user, I want to understand how the branch performance thresholds for better/worse are calculated and be able to verify and double check the calculation.